src/gui/executionlog.cpp

```diff
@@ -35,18 +35,22 @@
 #include <QPalette>
 #include "executionlog.h"
 #include "ui_executionlog.h"
-#include "base/logger.h"
+#include "base/preferences.h"
 #include "guiiconprovider.h"
 #include "loglistwidget.h"
 
 ExecutionLog::ExecutionLog(QWidget *parent)
     : QWidget(parent)
     , ui(new Ui::ExecutionLog)
-    , m_msgList(new LogListWidget(MAX_LOG_MESSAGES))
-    , m_peerList(new LogListWidget(MAX_LOG_MESSAGES))
+    , m_peerList(new LogListWidget(MAX_LOG_MESSAGES, true, true, true, true))
```

Comment:
To call widget classes in the form of  ```<Something>Widget``` (eg.  ExecutionLogWidget) is a good thing.